packages/web/src/app/notice/page.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+
+import { UseClientProvider } from "@sparcs-clubs/web/common/providers/UseClientProvider";
+
+import NoticePageMainFrame from "@sparcs-clubs/web/features/notice/frames/NoticePageMainFrame";
+
+const Clubs = () => (
+  <main>
+    <UseClientProvider>
+      <NoticePageMainFrame />
+    </UseClientProvider>
+  </main>
+);
+
+export default Clubs;
```

Comment:
저번에 `UseClientProvider`를 `layout.tsx`로 뻈어서 이제 매 페이지마다 추가하지 않으셔도 됩니다!!
추가로 export function name도 `Notice`로 바꿀게용

```suggestion
"use client";

import NoticePageMainFrame from "@sparcs-clubs/web/features/notice/frames/NoticePageMainFrame";

const Notice = () => <NoticePageMainFrame />;

export default Notice;
```

Reply:
확인했습니다!